data_management_basics/data_management_basics.md

```diff
@@ -1,14 +1,18 @@
 <!--
 author:   Ene Belleh
 email:    bellehe@chop.edu
-version: 1.0.1
-module_template_version: 3.0.0
+version: 1.1.0
+current_version_description: Fixed mermaidchart error that was causing diagram to not render; updated metadata
+module_type: standard
+docs_version: 2.0.0
 language: en
 narrator: US English Female
+mode: Textbook
 title: Research Data Management Basics
 comment: Learn the basics about research data management.
 long_description:  If you conduct research or work with research data or researchers, it's likely that research data management topics affect you.  Learn what research data management is, how to think about it in a structured way, and understand its scientific importance.
-estimated_time: 40 minutes
+estimated_time_in_minutes: 40
+pre_reqs:  The only pre-requisite suggested for this module is experience working in research in any capacity.
```

Comment:
```suggestion
@pre_reqs
The only pre-requisite suggested for this module is experience working in research in any capacity.
@end
```

The macros don't care if it is one line, but the processing scripts expect pre_reqs to be a block macro